server/server.go

```diff
@@ -470,6 +493,41 @@ func (s *Server) getRaft() *raftNode {
 	return r.(*raftNode)
 }
 
+// createActivityStream creates the activity stream and connects a local client
+// that will be subscribed to it.
+func (s *Server) createActivityStream() error {
+	if !s.config.ActivityStream.Enabled {
+		return nil
+	}
+
+	// Connect a local client that will be used to publish on the activity
+	// stream
+	listenAddress := s.config.GetListenAddress()
+	addrs := []string{listenAddress.Host + ":" + strconv.Itoa(s.port)}
+	var err error
+	s.activityStreamClient, err = lift.Connect(addrs)
+	if err != nil {
+		return errors.Wrap(err, "failed to connect the activity stream client")
+	}
+
+	options := []lift.StreamOption{
+		lift.Group(s.config.ActivityStream.Group),
+		lift.ReplicationFactor(s.config.ActivityStream.ReplicationFactor),
+		lift.Partitions(s.config.ActivityStream.Partitions),
```

Comment:
Should these be configurable? (We can always expose config for them at a future time)

Reply:
I have removed them for now in e4a135e6901aa6126086c69cb96bd31a6d73336a.